dropwizard-example/src/main/java/com/example/helloworld/resources/HelloWorldResource.java

```diff
@@ -39,10 +39,14 @@ public Saying sayHello(@QueryParam("name") Optional<String> name) {
     public void receiveHello(@Valid Saying saying) {
         LOGGER.info("Received a saying: {}", saying);
     }
-    
-    @GET
+
+    @POST
     @Path("/date")
-    public void receiveDate(@QueryParam("date") DateTimeParam dateTimeParam) {
-        LOGGER.info("Received a saying: {}", dateTimeParam.get());
+    public void receiveDate(@QueryParam("value") Optional<DateTimeParam> dateTimeParam) {
```

Comment:
It seems confusing to me that the path and the query parameter have the same name.
Let's leave it then, if it just my personal perception.